lib/validation/params.ex

```diff
@@ -0,0 +1,19 @@
+defmodule Litmus.Validation.Params do
+  @moduledoc false
+
+  @doc """
+  Module for checking if data consists of parameters that have not been defined in schema
+  """
+
+  @spec check_additional_params_error(map, map) :: tuple
+  def check_additional_params_error(data, schema) do
+    result = Map.keys(data) -- Map.keys(schema)
+
+    if result == [] do
+      {:ok, nil}
+    else
+      additional_params = Enum.join(result, ",")
+      {:error, "The data has following additional parameters: " <> additional_params}
```

Comment:
we should match an error message that Dora and lob-api currently respond. you can use the API to see what the messages are. in the case of a disallowed param beign passed, i get:

```
{
	"error": {
		"message": "asdf is not allowed",
		"status_code": 422
	}
}
```

So this string should be "asdf is not allowed" and only contain 1 parameter. It can be the first disallowed parameter found

Reply:
should we even return a status code?